openshift/openshift/src/main/java/io/undertow/protocols/spdy/SpdyChannelWithoutFlowControl.java

```diff
@@ -0,0 +1,45 @@
+package io.undertow.protocols.spdy;
```

Comment:
This should probably be in some 'org.arquillian.cube.openshift.impl....' package.